src/animation/AnimationAction.js

```diff
@@ -0,0 +1,665 @@
+/**
+ *
+ * Action provided by AnimationMixer for scheduling clip playback on specific
+ * objects.
+ *
+ * @author Ben Houston / http://clara.io/
+ * @author David Sarno / http://lighthaus.us/
+ * @author tschw
+ *
+ */
+
+THREE.AnimationAction = function() {
+
+	throw new Error( "THREE.AnimationAction: " +
+			"Use mixer.clipAction for construction." );
+
+};
+
+THREE.AnimationAction._new =
```

Comment:
How come we're using `_new` as constructor instead of `THREE.AnimationAction = function()`?